### ISSUE TITLE
Frequently vs Infrequently used Output Quantities

### ISSUE THREAD

**Issue by rmcdermo** [firemodels/fds#11064]
Kevin,
For v7, can we rework the output quantity lists?  I think that over time the original intent of these two tables has been lost.  I think we either need a a good cleaning of the "frequently used" table or else just to combine them.

**Comment by drjfloyd** [comment]
What about something like Gas, Particle, Solid, and HVAC? Something to help users more easily find outputs.

**Comment by rmcdermo** [comment]
I'm good with that.  Should then just be consistent with the various output routines in the source code.

**Comment by rpeacoc** [comment]
As a novice (and obviously coming at it from zone modeling viewpoint) where do fire quantities fit in? I can see parts in multiple groups. Should it have a group of its own?

**Comment by mcgratta** [comment]
There is a clear distinction between gas and solid phase. But the fire's HRR goes into a separate file, so the user never actually chooses that.

**Comment by drjfloyd** [comment]
For things like HRRPUV, HRRPUL, and layer quantities, I feel that those are clearly associated with the gas phase.

For some quantities there may be a bit of a judgement call as to where to best put them For example, AMPUA is a value measured at a wall but it involves a particle. IMO, this would be best put the particle quantity table. But for these I think a brief discussion about the general decision process should help users figure out which table to look in.

**Comment by mcgratta** [comment]
I made a first pass at reorganizing the output quantities. There are 5 tables -- gas, solid, devices/control/misc, particles, HVAC. There is still work to be done to explain various things as we have done before. For example, what do all those PRESSUREs mean. I will create a section to explain.. 

Also, I don't think we need to list all the outputs. Some of them are clearly for diagnostics and are not explained and might be misused. Things like DHDX look like they have been added for diagnostics only. But that is something to think about.
